server/resources/META-INF/cloudstack/core/spring-server-core-managers-context.xml

```diff
@@ -271,6 +271,6 @@
         class="org.apache.cloudstack.region.gslb.GlobalLoadBalancingRulesServiceImpl" >
         <property name="gslbServiceProviders" value="#{gslbServiceProvidersRegistry.registered}" />
     </bean>
-    <bean id="certServiceImpl" class="org.apache.cloudstack.network.lb.CertServiceImpl" />
+    <bean id="certServiceImpl" class="org.apache.cloudstack.network.ssl.CertServiceImpl" />
```

Comment:
could be but that would in this state be incorrect (see other discuss location in this review thread.